src/Apps/Home/Components/HomeEmergingPicksArtworksRail.tsx

```diff
@@ -23,9 +23,9 @@ interface HomeEmergingPicksArtworksRailProps {
   viewer: HomeEmergingPicksArtworksRail_viewer$data
 }
 
-export const HomeEmergingPicksArtworksRail: React.FC<React.PropsWithChildren<HomeEmergingPicksArtworksRailProps>> = ({
-  viewer,
-}) => {
+export const HomeEmergingPicksArtworksRail: React.FC<React.PropsWithChildren<
+  HomeEmergingPicksArtworksRailProps
+>> = ({ viewer }) => {
```

Comment:
`PropsWithChildren` should not be needed.

```suggestion
export const HomeEmergingPicksArtworksRail: React.FC<HomeEmergingPicksArtworksRailProps> = ({ viewer }) => {
```

Reply:
Hm, not sure why it's there, i didn't make this change

Reply:
I see we use PropsWithChildren in a lot of places. Will create a ticket to address it separately